src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -63,6 +63,8 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
     containerCssVars
   );
 
+  const ScheduleEventBookmarkClass = "ScheduleEvent__bookmark";
```

Comment:
can we move this static const out of the `ScheduleEvent` component not to create it on every render?
(would be better to see it right after the imports before the `ScheduleEventProps` interface)

Reply:
Done! 34b0660cab61d257db73fc7da72b9313eaa75589
Thanks! (and good to know how to creating a const on every render. It seems obvious now that you mention it, but hadn't realized it before.)